src/phoenix/server/api/mutations/prompt_version_tag_mutations.py

```diff
@@ -31,6 +32,7 @@ class SetPromptVersionTagInput:
 @strawberry.type
 class PromptVersionTagMutationPayload:
     prompt_version_tag: Optional[PromptVersionTag]
+    prompt: Optional[Prompt]
```

Comment:
Technically should always return this

Reply:
do you want the delete mutation to always return the prompt as well?